types/params.go

```diff
@@ -79,9 +83,9 @@ func (params *ConsensusParams) Validate() error {
 	}
 
 	// ensure blocks aren't too big
-	if params.BlockSizeParams.MaxBytes > maxBlockSizeBytes {
+	if params.BlockSizeParams.MaxBytes > _100MiB {
```

Comment:
this isn't good because if we change the limit we have to change the variable name. it really should just be maxBlockSizeBytes

Reply:
Great point here, thanks.